packages/services/document-service/src/main/java/software/uncharted/terarium/documentserver/proxies/xdd/DocumentProxy.java

```diff
@@ -47,4 +47,12 @@ Response getRelatedDocuments(
 		@PathParam("set") String set,
 		@QueryParam("docid") String docid
 	);
+
+	@GET
+	@Path("/sets/{set}/word2vec/api/most_similar")
+	Response getRelatedWords(
```

Comment:
There is a recent update to the document-service to remove all of the `Response` return types in favour of proper serialization of java objects.  If you merge main into your branch you'll see how this is being done.  `getRelatedWords` should return a java object here as a response